Doppler.ContactPolicies.Data.Access/Repositories/ContactPoliciesSettings/ContactPoliciesSettingsRepository.cs

```diff
@@ -0,0 +1,52 @@
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Dapper;
+using Doppler.ContactPolicies.Data.Access.Core;
+using Doppler.ContactPolicies.Data.Access.Entities;
+
+namespace Doppler.ContactPolicies.Data.Access.Repositories.ContactPoliciesSettings
+{
+    public class ContactPoliciesSettingsRepository : IContactPoliciesSettingsRepository
+    {
+        private readonly IDatabaseConnectionFactory _databaseConnectionFactory;
+
+        public ContactPoliciesSettingsRepository(IDatabaseConnectionFactory databaseConnectionFactory)
+        {
+            _databaseConnectionFactory = databaseConnectionFactory;
+        }
+
+        #region public methods
+
+        public async Task<Entities.ContactPoliciesSettings> GetContactPoliciesSettingsAsync(string accountName)
+        {
+            using var connection = await _databaseConnectionFactory.GetConnection();
+            var query =
+                @"select cp.IdUser, cp.Enabled [EnabledExcludedSubscribersList], cp.Active, cp.Interval [IntervalInDays], cp.Amount [EmailsAmountByInterval], u.Email [AccountName]
+                from [User] u
+                inner join [UserShippingLimit] cp on u.IdUser = cp.IdUser where u.Email = @Email;
```

Comment:
Why not simply filter by `Enabled`? in that way you will not need the field `EnabledExcludedSubscribersList`.

```suggestion
                @"select cp.IdUser, cp.Active, cp.Interval [IntervalInDays], cp.Amount [EmailsAmountByInterval], u.Email [AccountName]
                from [User] u
                inner join [UserShippingLimit] cp on u.IdUser = cp.IdUser 
                where u.Email = @Email and cp.Enabled = 1;
```

Reply:
if i do that them i couldn't know if the user exist or simply dont have the permissions for excluder subscribers list and them the message returned to the user should change to "No contact policies settings for account name: {accountName}"